de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/ui/browser/EclipseBrowserDialog.java

```diff
@@ -46,4 +47,25 @@ private void centerShellRelativeToParent() {
             + (parentShellBounds.width - shellSize.x) / 2, parentShellBounds.y
             + (parentShellBounds.height - shellSize.y) / 2);
     }
+
+    private void sizeShellNotBiggerThanParent() {
```

Comment:
Why not `sizeShellSmallerThanParent` instead of the negated version?

Reply:
The name still seems overly "wordy" to me. My suggestion would be to call it `adjustShellSize` and add a more detailed explanation as javadoc for the method.